test/NodeRegistry.t.sol

```diff
@@ -46,7 +46,7 @@ contract NodeRegistryTests is Test, Utils {
     /* ============ initial state ============ */
 
     function test_initialState() public view {
-        assertEq(registry.maxActiveNodes(), 20);
+        assertEq(registry.getMaxActiveNodes(), 20);
```

Comment:
Generally, argument-less getters do no need the `get` prefix (i.e. `totalSupply`, `symbol`, etc).